### DISCUSSION TITLE
Am I supposed to call other methods before or after calling play()?

### DISCUSSION THREAD

**Discussion by MattTreichelYeah** [goldfire/howler.js#1562] [Q&A] [answered]
It seems like a lot of the examples first call `play()`, and then methods like `rate()`, `volume()`, or `seek()`. Is it required to do it in this order? Logically I would think I want to prepare the sound and then call `play()` so that there's no possible "blip" of adjusting mid-sound, but, I'm confused about how Howler manages multiple instances of a sound. If I call `play()` multiple times in a loop like so, 

```
do {
  sound.seek(<value>);
  sound.volume(<value>);
  sound.rate(<value>);
  sound.play();
}
```

would usages past the first loop actually apply to the previous instance of the sound rather than the one I invoke in the current iteration? How does it work on first loop, is it special behaviour to preset the values before any sound instance is `play()`ed?

**Comment by goldfire** [comment, marked as answer]
Yes, you can call those before and howler will queue those for you. However, if you were playing multiple sounds, calling them before without a sound ID is going to affect all current and future sounds from that howl instance, rather than just the sound you are about to play. So, if you want it to affect the sound you are about to play, you need to pass the id and call them after you call play.

**Reply by MattTreichelYeah** [reply]
Thanks, I think that helps me understand some of what was happening in https://github.com/goldfire/howler.js/issues/1556, I'll close it. Good to know the properties affect all instances without an id.
